src/callbacks_step/analysis.jl

```diff
@@ -496,6 +496,27 @@ function print_amr_information(callbacks, mesh::P4estMesh, solver, cache)
   return nothing
 end
 
+# Print level information only if AMR is enabled
+function print_amr_information(callbacks, mesh::T8codeMesh, solver, cache)
+
+  # Return early if there is nothing to print
+  uses_amr(callbacks) || return nothing
+
+  levels = trixi_t8_get_local_element_levels(mesh.forest)
+
+  min_level = minimum(levels)
+  max_level = maximum(levels)
+
+  mpi_println(" minlevel = $min_level")
+  mpi_println(" maxlevel = $max_level")
+
+  # for level = max_level:-1:min_level+1
+  #   mpi_println(" ├── level $level:    " * @sprintf("% 14d", elements_per_level[level + 1]))
+  # end
+  # mpi_println(" └── level $min_level:    " * @sprintf("% 14d", elements_per_level[min_level + 1]))
+
```

Comment:
What about this code in comments?

Reply:
This will be used when MPI is working for `T8codeMesh`. Will add a TODO.